Library/Homebrew/cask/cask.rb

```diff
@@ -281,6 +281,19 @@ def to_hash_with_variations
 
     private
 
+    def artifacts_list
+      artifacts.map do |artifact|
+        if artifact.is_a? Artifact::AbstractFlightBlock
+          { type: artifact.summarize }
```

Comment:
Are these useful to have at all?

Reply:
Well, it depends on how you look at it. No, they won't be used directly in cask loading, but they can be used as an indicator that there _are_ flight blocks that won't be loaded. So depending on how we choose to handle these later on this may be useful or not at all. Plus, they were present in the existing artifact hash.